### ISSUE TITLE
How to think about the loss normaliztion

### ISSUE THREAD

**Issue by gujiaqivadin** [hekmak/Conf-net#1]
https://github.com/hekmak/Conf-net/blob/ea14103d3fd83dba288be1166c3f55a69a949013/main/depth_completion.py#L202

Hello, hekmak!
     Thanks for you great code for depth completion error map. And the results are great. I am very interested in this idea. And when I read your code, I find that the loss normalization to balance the loss of depth and loss of error is very crucial. But I can't understand the baseline loss meanings. In your tensorflow code about loss normalization, loss = 1/1 + 1/1 = 2. Then how to train the net indeed. I am a little confused about what is exactly the baseline loss represented for.

**Comment by hekmak** [comment]
Hi,
Every iteration both losses get normalized based on their values. This is mostly for equal weighting between two losses, although the normalization stabilizes the training. You can read the paper for more info.
